uaclient/cli.py

```diff
@@ -222,6 +224,32 @@ def attach_parser(parser):
     return parser
 
 
+def fix_parser(parser):
+    """Build or extend an arg parser for fix subcommand."""
+    parser.usage = USAGE_TMPL.format(
+        name=NAME, command="fix <CVE-yyyy-nnnn>|<USN-nnnn>"
+    )
+    parser.prog = "fix"
+    parser._optionals.title = "Flags"
+    parser.add_argument(
+        "security_issue",
+        help=(
+            "Security vulnerability ID to inspect and resolve on this system."
+            " Format: CVE-yyyy-nnnn or USN-nnnn"
```

Comment:
I think we should update the CVE format, since it can accept formats like  `CVE-yyyy-nnnnn`  and `CVE-yyyy-nnnnnn`

Reply:
I think this is still an issue